### ISSUE TITLE
v0.15.1 windows & vlc

### ISSUE THREAD

**Issue by skaox** [4gray/iptvnator#369]
**Describe the bug**

Need to put vlc path
C:\VLC\vlc.exe
**for mpv & vlc**
if you don't it will not start

**Default vlc path**  C:\Program Files\VideoLAN\VLC\ **not working**
**Installation** -> **Advanced** -> C:\VLC\  **or**  C:\VideoLAN\VLC\

 - OS: Windows 10/11 x64
 - Version 0.15.1

**Additional context**

Already reported in v0.15.0

**THANK YOU** now series are working with xtreme code & vlc

**Comment by skaox** [comment]
Ubuntu Mate 20.04 (appimage) & 22.04 (deb)
nothing to report about that they're working without it.
What's the result of
`which mpv vlc`

**Comment by spike1986** [comment]
![Bildschirmfoto vom 2024-04-05 07-31-27](https://github.com/4gray/iptvnator/assets/10047537/202989e9-0329-41a0-8772-2ae7c067e84a)
Thanks for your quick response.
I have installed Ubuntu 22.04.4 LTS (gnome).
Here is my which. I have also specified at IPTVnator

**Comment by skaox** [comment]
Did you found a solution ? Not working with /snap/bin/vlc ?

**Uninstall snap version and try the classic ppa deb.**

(-: Another example with chromium **snap** and an add-on where
NodeJS or BUN wasn't detected no problem with the other one ;-)

**Comment by codevski** [comment]
The work around for windows still doesnt work. Moved the installation to C:\VLC\vlc.exe still no go. Where are the logs located?

**Comment by olstern01** [comment]
Not working for me as well.

**Comment by skaox** [comment]
Just in case it wasn't clear in first post you need to put vlc path twice **one time in mpv path and another time for vlc**
if you don't do that sure it will not work.

**Comment by skaox** [comment]
(-: Well done exactly first screenshot mpv was the selected player instead of VLC.

Windows is for family/friends or like it's name in a window I'm using iptvnator with linux .and xtrem code..
My provider mpv will work nicely but only with TV for VOD VLC is the only one who's accepted without any problem.

Now there is also a [webplayer](https://github.com/lKinderBueno/Streamity-Xtream-IPTV-Web-player) involved with a well done nav interface but need [chromium with all codecs](https://chromium.woolyss.com/).

Possible to add one [extension/plugin for opening VOD with VLC](https://chromewebstore.google.com/detail/open-in-vlc-media-player/ihpiinojhnfhpdmmacgmpoonphhimkaj) who need [native-client/NodeJS](https://github.com/andy-portmen/native-client)  because
webplayer can't manage mkv multi audio or sub same thing for AC3 not supported by web navigators ;-)

**Comment by enerdude** [comment]
Yeah trick here seems to be to enter the path under mpv player as well, even if you just want to use vlc player.
If you use the default installation path of vlc, use the following:

"C:\Program Files\VideoLAN\VLC\vlc.exe"
or
C:\Progra~1\VideoLAN\VLC\vlc.exe

under both mpv path and vlc path and then set player to vlc
